packages/fxa-content-server/app/scripts/models/polls/device-connected.js

```diff
@@ -0,0 +1,100 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Backbone from 'backbone';
+import { VERIFICATION_POLL_IN_MS } from '../../lib/constants';
+
+export default class DeviceConnectedPoll extends Backbone.Model {
+  constructor(data, options = {}) {
+    super(options, data);
+
+    if (!options.account) {
+      throw new Error('options.account required');
+    }
+
+    this._account = options.account;
+    this._pollIntervalInMS =
+      options.pollIntervalInMS || VERIFICATION_POLL_IN_MS;
```

Comment:
I wonder if it makes sense to add a new constant instead of reusing the `VERIFICATION_POLL_IN_MS`. The device call could end up expensive (it used to be in the past) and we might need to change that value

Reply:
Updated!